buildout.cfg

```diff
@@ -13,6 +13,10 @@ recipe = zc.recipe.egg
 eggs =
     Sphinx
     docutils
+    ZODB
+    j1m.sphinxautointerface
```

Comment:
I think :members: support was added in 0.6:

  0.6 (2011-01-28)
  Correctly handle ‘:members:’ values explicitly set in the directive.